smartsim/_core/control/controller.py

```diff
@@ -298,35 +310,43 @@ def _launch(self, manifest: Manifest) -> None:
         :param manifest: Manifest of deployables to launch
         :type manifest: Manifest
         """
-        orchestrator = manifest.db
-        if orchestrator:
+
+        orchestrators = manifest.dbs
+        # Loop over deployables to launch and launch multiple orchestrators
+        for orchestrator in orchestrators:
+            for key in self._jobs.get_db_host_addresses():
+                _, db_id = unpack_db_identifier(key, "_")
+                if orchestrator.name == db_id:
+                    raise DBIDConflictError(
+                        f"Database identifier {orchestrator.name}"
+                        " has already been used. Pass in a unique"
+                        " name for db_identifier"
+                    )
+
             if orchestrator.num_shards > 1 and isinstance(
                 self._launcher, LocalLauncher
             ):
                 raise SmartSimError(
                     "Local launcher does not support multi-host orchestrators"
                 )
-            if self.orchestrator_active:
-                msg = "Attempted to launch a second Orchestrator instance. "
-                msg += "Only 1 Orchestrator can be active at a time"
-                raise SmartSimError(msg)
             self._launch_orchestrator(orchestrator)
 
         if self.orchestrator_active:
             self._set_dbobjects(manifest)
 
         # create all steps prior to launch
         steps: t.List[t.Tuple[Step, t.Union[SmartSimEntity, EntityList]]] = []
+
         all_entity_lists = manifest.ensembles
         for elist in all_entity_lists:
             if elist.batch:
                 batch_step = self._create_batch_job_step(elist)
                 steps.append((batch_step, elist))
             else:
-                # if ensemble is to be run as separate job steps, aka not in a batch
+                # if ensemble is to be run as separate job steps,
```

Comment:
This comment is slightly "redundant", as it simply explains what `.batch` means. Maybe it is left over from debugging?